components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.swift

```diff
@@ -0,0 +1,183 @@
+
+    let infoPage = UIView(frame: CGRect())
+    infoPage.translatesAutoresizingMaskIntoConstraints = false
+    infoPage.backgroundColor = MDCPalette.lightBlue().tint300
+    scrollView.addSubview(infoPage)
+
+    let infoLabel = UILabel(frame: CGRect())
+    infoLabel.translatesAutoresizingMaskIntoConstraints = false
+    infoLabel.textColor = UIColor.white
+    infoLabel.numberOfLines = 0
+    infoLabel.text = "Tabs enable content organization at a high level, such as switching between views"
+    infoPage.addSubview(infoLabel)
+
+    // Layout the views to be equal height and width to each other and self.view, hug the edges of the
+    // scrollView and meet in the middle.
+
+    NSLayoutConstraint(item: infoLabel, attribute: .centerX, relatedBy: .equal, toItem: infoPage, attribute: .centerX, multiplier: 1, constant: 0).isActive = true
+    NSLayoutConstraint(item: infoLabel, attribute: .centerY, relatedBy: .equal, toItem: infoPage, attribute: .centerY, multiplier: 1, constant: -50).isActive = true
+
+    NSLayoutConstraint.activate(NSLayoutConstraint.constraints(withVisualFormat: "H:|-50-[infoLabel]-50-|",
+                                                               options: [],
+                                                               metrics: nil,
+                                                               views: ["infoLabel": infoLabel]))
+    NSLayoutConstraint(item: infoPage, attribute: .width, relatedBy: .equal, toItem: self.view, attribute: .width, multiplier: 1, constant: 0).isActive = true
+    NSLayoutConstraint(item: infoPage, attribute: .height, relatedBy: .equal, toItem: self.scrollView, attribute: .height, multiplier: 1, constant: 0).isActive = true
+
+    NSLayoutConstraint(item: self.starPage, attribute: .width, relatedBy: .equal, toItem: infoPage, attribute: .width, multiplier: 1, constant: 0).isActive = true
+
+    let views = ["infoPage": infoPage, "starPage": self.starPage]
+
+    NSLayoutConstraint.activate(NSLayoutConstraint.constraints(withVisualFormat: "H:|[infoPage][starPage]|", options: [.alignAllTop, .alignAllBottom], metrics: nil, views: views))
+    NSLayoutConstraint.activate(NSLayoutConstraint.constraints(withVisualFormat: "V:|[infoPage]|", options: [], metrics: nil, views: views))
+
+    addStar(centered: true)
+  }
+
+  func setupStarPage() -> UIView {
+    let starPage = UIView(frame: CGRect())
+    starPage.translatesAutoresizingMaskIntoConstraints = false
+    starPage.backgroundColor = MDCPalette.lightBlue().tint200
+    self.scrollView.addSubview(starPage)
+
+    return starPage
+  }
+
+  func addStar(centered: Bool) {
+    let starImage = UIImage(named:"TabBarDemo_ic_star", in:Bundle(for: type(of: self)), compatibleWith:nil)
+    let starView = UIImageView(image: starImage)
+    starView.translatesAutoresizingMaskIntoConstraints = false
+    starPage.addSubview(starView)
+    starView.sizeToFit()
+
+    let x = centered ? 1.0 : (CGFloat(arc4random_uniform(199)) + 1.0) / 100.0 // 0 < x <=2
```

Comment:
I feel like this might gain a little readability with parens around the second branch of the ternary - not 100% clear where the "/ 100.0" gets grouped